types/static-kdtree/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+      "esModuleInterop": true,
```

Comment:
```suggestion
```
not sure why this appeared here now? `esModuleIntertop` is no-no for declarations file here (why: https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports)
I'll update other code to reflect this removal if needed

Reply:
Umm...interesting to know. This is tied to importing typings of `ndarray`. Its definition presents itself as a cjs module. My editor complained the import line. Thus this extra config was added.

As the line was changed to cjs import, this config can be dropped. Thanks.

Just out of curiosity